src/commands/get/index.ts

```diff
@@ -49,19 +56,41 @@ export default class FlagsmithGet extends Command {
     }
 
     const output = flags.output
+    const entity = flags.entity
+    const isDocument = entity === 'document'
+
+    if (isDocument && !environment?.startsWith('ser.')) {
+      this.error('In order to fetch the environment document you need to provide a server-side SDK token.')
+      return
+    }
+
     outputString += `, outputing to ${output}.`
     this.log(outputString)
 
-    await flagsmith.init({
-      environmentID: environment,
-      fetch: fetch,
-      api: api,
-      identity: identity,
-    })
-    if (flags.pretty) {
-      fs.writeFileSync(output, JSON.stringify(flagsmith.getState(), null, 2))
+    if (environment?.startsWith('ser.')) {
+      fetch(`${api || 'https://edge.api.flagsmith.com/api/v1/'}environment-document/flags/`, {
```

Comment:
Not sure what this URL is? I think it should just be: 

```suggestion
      fetch(`${api || 'https://edge.api.flagsmith.com/api/v1/'}environment-document/`, {
```

Reply:
Weird, interestingly it still works. Have adjusted

Reply:
Ah, interesting. It's possibly just the way in which pattern matching works in lambda / serverless. Probably needs a `$` at the end of the pattern or something.